src/common/utils.xsl

```diff
@@ -336,25 +341,42 @@
         <xsl:sequence select="fn:concat(upper-case(substring($cc, 1, 1)), substring($cc, 2))"/>
     </xsl:function>
 
-
-
+    <xd:doc>
+        <xd:desc>Smooth a string</xd:desc>
+        <xd:param name="str">input string to be smoothed</xd:param>
+    </xd:doc>
+    <xsl:function name="f:smooth-string" as="xs:string?">
+        <xsl:param name="str" as="xs:string"/>
+        <xsl:sequence select="
+            if ($str = 'Has u r l') then
```

Comment:
We cannot do this because we do not know which Acronyms are used in the model. 
 
We need to detect sequences of consecutive upper-case characters and insert a space only between the last two.